### PR TITLE
[CI Unit Test in progress] Feature integrate android 10-X

### PR DESCRIPTION
**Pull request by GuillaumeSE**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [ ] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/ReleaseNotes/_ReleaseNotes.md [hunk 1]
```diff
@@ -142,18 +142,7 @@
...
-* XAML Hot Reload support for iOS, Android and Windows
-* Add support for GitPod Workspace and prebuilds
-* #880 Added added implicit conversion for double to Thickness
-* Add Android support for `CoreApplication.GetCurrentView().TitleBar.ExtendViewIntoTitleBar` to programatically draw under the status bar
-* [WASM] `ScrollViewer.ChangeView` is now supported
-* [Wasm] Add the ability to focus a TextBox by clicking its header
-* Add support for `ToggleButton.IsThreeState` and `ToggleButton.Indeterminate`
-* [Wasm] Add support for `TextBox.IsReadonly`
-* [iOS] [WASM] `Path` now supports `LinearGradientBrush` as `Fill`
-* A feature flag has been added to change the default preferred placement mode fo the drop down of the `ComboBox` (cf. ../articles/control/ComboBox.md)
```
Comment (carldebilly) [inline doc/ReleaseNotes/_ReleaseNotes.md]: You deleted release notes, take care.

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 2]
```diff
@@ -28,7 +27,7 @@ Make sure that the version number is the same as the `Uno.UI` package.
 ## Troubleshooting
 - The application logs file reloads, so you should see diagnostics messages when a XAML file is reloaded.
 - The output window in VS has an output named "Uno Platform" in its drop down. Diagnostics messages from the VS integration appear there.
-- The file named `obj\Debug\XXX\g\RemoteControlGenerator\RemoteControl.g.cs` contains the connection information, verify that the information makes sense, particularly the port number.
+- The file named `obj\Debug\XXX\g\RemoteControlGenerator\HotReload.g.cs` contains the connection information, verify that the information makes sense, particularly the port number.
```
Comment (carldebilly) [inline doc/articles/features/working-with-xaml-hot-reload.md]: You rolled-back that change

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 3]
```diff
@@ -10,12 +10,11 @@ The Uno Platform Hot Reload feature provides a way to modify the XAML displayed
 - Cross-platform Hot Reload is supported
 
 ## How to use the XAML Hot Reload
-- Install the [Uno Platform Add-in](https://marketplace.visualstudio.com/items?itemName=nventivecorp.uno-platform-addin) from the Visual Studio Marketplace.
 - Create a sample application using the **Uno Cross Platform App** template
 - Build an Uno application head (iOS, Android or WebAssembly), start it (with or without the debugger)
 - Change a XAML file from VS and the app should update
 
-> Note: You can also get pre-released version of the Visual Studio Extension from the [master CI builds artifacts](https://dev.azure.com/uno-platform/Uno%20Platform/_build?definitionId=5&_a=summary). 
+> Important note: If you're using this feature and an Uno stable release containing it has not been published, you will need to get the Visual Studio Extension from the [master CI builds artifacts](https://dev.azure.com/uno-platform/Uno%20Platform/_build?definitionId=5&_a=summary).
```
Comment (carldebilly) [inline doc/articles/features/working-with-xaml-hot-reload.md]: You rolled-back that change

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 4]
```diff
@@ -39,8 +38,7 @@ Make sure that the version number is the same as the `Uno.UI` package.
     - Resolution: Restart VS and rebuild the app
 - The reload server may start twice (The VS **Uno Platform** output window shows two "Starting server" messages)
     - Resolution: Restart visual studio, rebuild the app.
-- The app does not update its XAML, because the port number in `RemoteControl.g.cs` is `0`.
-    - Ensure you have the right version of the _VSIX_ installed.
+- The app does not update its XAML, because the port number in `HotReload.g.cs` is `0`.
```
Comment (carldebilly) [inline doc/articles/features/working-with-xaml-hot-reload.md]: Change roll-back

File: src/AddIns/Uno.UI.Lottie/LottieVisualSource.Android.cs [hunk 5]
```diff
@@ -29,7 +29,7 @@ public void Update(AnimatedVisualPlayer player)
 			{
 				_animation = new LottieAnimationView(Android.App.Application.Context);
 				_animation.EnableMergePathsForKitKatAndAbove(true);
-				_animation.UseHardwareAcceleration(UseHardwareAcceleration);
+				_animation.SetRenderMode(RenderMode.Hardware);
```
Comment (carldebilly) [inline src/AddIns/Uno.UI.Lottie/LottieVisualSource.Android.cs]: Is it related to Android 10X ?

File: src/SamplesApp/SamplesApp.UITests/SampleControlUITestBase.cs [hunk 6]
```diff
@@ -145,9 +145,7 @@ public FileInfo TakeScreenshot(string stepName, ScreenshotOptions options)
 			var fileNameWithoutExt = Path.GetFileNameWithoutExtension(fileInfo.Name);
 			if (fileNameWithoutExt != title)
 			{
-				var destFileName = Path
-					.Combine(Path.GetDirectoryName(fileInfo.FullName), title + Path.GetExtension(fileInfo.Name))
-					.GetNormalizedLongPath();
+				var destFileName = Path.Combine(Path.GetDirectoryName(fileInfo.FullName), title + Path.GetExtension(fileInfo.Name));
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/SampleControlUITestBase.cs]: Change roll-back

File: src/SamplesApp/SamplesApp.UITests/SampleControlUITestBase.cs [hunk 7]
```diff
@@ -167,9 +165,7 @@ public FileInfo TakeScreenshot(string stepName, ScreenshotOptions options)
 
 			if(options != null)
 			{
-				var fileName = Path
-					.Combine(fileInfo.DirectoryName, Path.GetFileNameWithoutExtension(fileInfo.FullName) + ".metadata")
-					.GetNormalizedLongPath();
+				var fileName = Path.Combine(fileInfo.DirectoryName, Path.GetFileNameWithoutExtension(fileInfo.FullName) + ".metadata");
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/SampleControlUITestBase.cs]: Change roll-back

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs [hunk 8]
```diff
@@ -35,19 +34,5 @@ public void When_TransformToVisual_ScrollViewer()
...
-		{
-			Run("UITests.Shared.Windows_UI_Xaml.UIElementTests.UIElement_Native_Child");
-
-			_app.WaitForElement("SpacerBorder");
-			var spacerRect = _app.GetRect("SpacerBorder");
-
-			var scrn = TakeScreenshot("Ready");
-
-			ImageAssert.HasColorAt(scrn, spacerRect.X, spacerRect.Y, Color.Red);
-		}
```
Comment (carldebilly) [inline src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml/UIElementTests/UIElementTests.cs]: You removed a test

**Comment by GuillaumeSE** [comment]
Rework have been done in
https://github.com/unoplatform/uno/pull/2549

File: src/AddIns/Uno.UI.Lottie/LottieVisualSource.Android.cs (hunk 5 above)
Comment (GuillaumeSE) [reply src/AddIns/Uno.UI.Lottie/LottieVisualSource.Android.cs]: this is due to the new lottie version 3.0.4 that support Android X

**Pull request closed without merging**